### PR TITLE
Update README to be clearer.

### PR DESCRIPTION
**Pull request by Hixie**
cc @CaseyHillers @godofredoc

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,94 +1,158 @@
...
-You can test the new version by accessing `{VERSION}-dot-flutter-dashboard.appspot.com` in your
-browser. If the result is satisfactory, the new version can be activated by using the Cloud Console
-UI: https://pantheon.corp.google.com/appengine/versions?project=flutter-dashboard&serviceId=default
+## Forcing a refresh from GitHub
 
-## Optional flags
+The server is driven by commits made to
+https://github.com/flutter/flutter repo. It periodically syncs new
+commits. If you need to manually force a refresh, query
+`https://flutter-dashboard.appspot.com/api/refresh-github-commits`.
```
Comment (CaseyHillers) [inline README.md]: This is only allowed for users that have auth access to Cocoon (Googlers and whitelisted accounts)

**Comment by CaseyHillers** [comment]
Rebasing with master should fix the issue LUCI is throwing.

File: README.md [hunk 2]
```diff
@@ -1,94 +1,158 @@
...
+Dart SDK, we just use the command line tools.
 
-## Forcing a refresh from GitHub
 
-Cocoon is driven by commits made to https://github.com/flutter/flutter repo. It
-periodically syncs new commits. If you need to manually force a refresh, query
-`https://flutter-dashboard.appspot.com/api/refresh-github-commits`.
+## Developing the server
+
+The server depends on some Google-internal features, and so can only
```
[the remark this answers is not in the document]
Comment (Hixie) [reply README.md]: What is the bit about the GCP dashboard about then?

[the remark this answers is not in the document]
Comment (CaseyHillers) [reply README.md]: With the Dart server, I have only needed the GCP dashboard for managing the deployments. The old Go dev server environment used to allow forwarding for Googlers to existing AppEngine services, however, this has not been implemented in the Dart dev server environment. Tracking issue at https://github.com/flutter/flutter/issues/45410

**Comment by Hixie** [comment]
Updated the PR for the comments to which I gave a :+1: .

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Hixie) [reply README.md]: ok I tweaked the text a bit and i'll let you and @keyonghan update it. :-)

**Comment by Hixie** [comment]
Once the tests go green I'll land this. Thanks!

**Pull request merged**